src/styles/blocks/button.scss

```diff
@@ -0,0 +1,27 @@
+.button {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+
+  font-weight: 700;
+  text-decoration: none;
+
+  height: 56px;
+  width: 100%;
+
+  font-family: inherit;
```

Comment:
without an explicit indication for some reason does not inherit the font from .page (maybe because i use <button> tag, now i changed tag to div, and inherit is working